2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -134,39 +134,41 @@ function replset_addr() {
 }
 
 # mongo_remove removes the current MongoDB from the cluster
+# optionally takes address of replset to connect and address of server to remove
 function mongo_remove() {
   local host
   # if we cannot determine the IP address of the primary, exit without an error
   # to allow callers to proceed with their logic
```

Comment:
OK. Substituted address to identifier.

IMPOV, "address of replset" is not "address of primary member of replset". It some identifier how to connect to replset.